task/buildah-min/0.4/buildah-min.yaml

```diff
@@ -1,3 +1,4 @@
+# WARNING: This is an auto generated file, do not modify this file directly
```

Comment:
I think we can remove the comment, no need to add buildah-min to this PR

Reply:
If I remove it, CI fails.
It doesn't belong in this PR, but unless I do it on another PR, this PR won't pass CI.

Not sure why this is showing now...

Reply:
OK, I understand.

Reply:
I ended up [creating a PR](https://github.com/konflux-ci/build-definitions/pull/2032) for that as it affected also our tekton-tools update PR.